src/app/tree-view/tree-node-selection/tree-node-selection.html

```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+<clr-tree [(clrTreeSelected)]="selection">
```

Comment:
Nope. In this example we toggle a node using the `clrTreeNodeSelected` input. This example is going to be updated though.